clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/description/servergroup/DeployKubernetesAtomicOperationDescription.groovy

```diff
@@ -375,3 +376,27 @@ class KubernetesSeLinuxOptions {
   String type
   String level
 }
+
+@AutoClone
+@Canonical
+//Base on model https://kubernetes.io/docs/resources-reference/v1.5/#podspec-v1
+//which will map to Kubernetes 1.5 API version
+class KubernetesPodSpecDescription {
+  Long activeDeadlineSeconds
+  List<KubernetesContainerDescription> containers
+  String dnsPolicy
+  Boolean hostIPC
+  Boolean hostNetwork
+  Boolean hostPID
+  String hostname
```

Comment:
Controller will follow PodTemplateSpec which will be using PodSpec.  https://kubernetes.io/docs/resources-reference/v1.5/#podtemplatespec-v1